src/commands/synthetics/utils.ts

```diff
@@ -545,12 +553,24 @@ export const getTestAndOverrideConfig = async (
   }
   reporter.testWait(test)
 
-  if (isTunnelEnabled && test.type !== 'browser' && test.subtype !== 'http') {
+  if (isTunnelEnabled && !isTestSupportedByTunnel(test)) {
+    const details = [`public ID: ${test.public_id}`, `type: ${test.type}`]
+
+    if (test.subtype) {
+      details.push(`sub-type: ${test.subtype}`)
+    }
+
+    if (test.subtype === 'multi') {
+      const unsupportedStepSubTypes = (test.config.steps || [])
+        .filter((step) => step.subtype !== 'http')
+        .map(({subtype}) => subtype)
+
+      details.push(`step sub-types: [${unsupportedStepSubTypes.join(', ')}]`)
+    }
+
     throw new CriticalError(
       'TUNNEL_NOT_SUPPORTED',
-      `The tunnel is only supported with HTTP API tests and Browser tests (public ID: ${test.public_id}, type: ${
-        test.type
-      }${test.subtype ? `, sub-type: ${test.subtype}` : ''}).`
+      `The tunnel is only supported with HTTP API tests and Browser tests (${details.join(', ')}).`
```

Comment:
I kept this error message because a multi-step test is a subtype of API test so in a sense, HTTP steps are HTTP API tests.